ddtrace/contrib/asgi/middleware.py

```diff
@@ -120,9 +124,14 @@ async def __call__(self, scope, receive, send):
             trace_utils.activate_distributed_headers(
                 self.tracer, int_config=self.integration_config, request_headers=headers
             )
-        resource = " ".join((scope["method"], scope["path"]))
-        operation_name = self.integration_config.get("request_span_name", "asgi.request")
-        operation_name = schematize_url_operation(operation_name, direction=SpanDirection.INBOUND, protocol="http")
+        resource = " ".join([method, scope["path"]])
+        if scope["type"] == "http":
+            operation_name = self.integration_config.get("request_span_name", "asgi.request")
+            operation_name = schematize_url_operation(operation_name, direction=SpanDirection.INBOUND, protocol="http")
+        else:
+            # in the case of websockets we don't currently schematize the operation names
+            operation_name = self.integration_config.get("request_span_name", "asgi.request")
```

Comment:
This is a duplicate of line 129. Deduplicating them would reduce the potential for future bugs in this code.